src/docs/manual/02_install.adoc

```diff
@@ -55,66 +53,100 @@ After unzipping, put the dependencies in `resources` folder, so that the structu
+
+- DE - German.
+- EN - English.
+- ES - Spanish.
+
+.Linux initArc42EN example
 [source, bash]
 ----
-./mvnw
+cd <docToolchain home>
+./gradlew -b init.gradle initArc42EN -PdocDir=<your directory>
 ----
 
-.Windows with gradle wrapper
+The Config.groovy file is then preconfigured to use the downloaded template.
+
+== Build
+
+This should already be enough to start a first build. Due to some limitations in gradle, you need to run the build from
+the docToolchain home directory, and point docDir property to an absolute directory where your documents live.
+Alternatively you could use helper script `docs.sh` that is alongside `Config.groovy`.
+
+.Linux with gradle wrapper
 [source, bash]
 ----
-./gradlew
+./gradlew -PdocDir=<your absolute directory>
 ----
 
-.Windows with maven wrapper
+.Windows with gradle wrapper
 [source, bash]
 ----
-./mvnw.bat
+gradlew -PdocDir=<your absolute directory>
 ----
 
 As a result, you will see the progress of your build together with some warnings which you can just ignore for the moment.
 
-The first build generated some file within the `<project>/build` folder:
+The first build generated some files within the `<your absolute directory>/build`:
 
 [source]
 ----
-<project>:
-    build:
-        docs:
-            html5:
-                images:
-                arc42-template-de.html
-                arc42-template-end.html
-            pdf:
-                arc42-template-de.pdf
+build
+|-- html5
+|   |-- arc42-template.html
+|   `-- images
+|       |-- 05_building_blocks-EN.png
+|       |-- 08-Crosscutting-Concepts-Structure-EN.png
+|       `-- arc42-logo.png
+`-- pdf
+    |-- arc42-template.pdf
+    `-- images
+        |-- 05_building_blocks-EN.png
+        |-- 08-Crosscutting-Concepts-Structure-EN.png
+        `-- arc42-logo.png
 ----
 
 *Congratulations!* if you see the same folder structure, you just managed to render the standard arc42 template as html and pdf!
 
 If you didn't get the right output, please raise an issue on https://github.com/docToolchain/docToolchain/issues[github]
 
-== Custom configuration
+== Publish on Confluence
 
-You can install docToolchain in a directory separate from your project. Then docToolchain is a standalone tool that can be
-upgraded independently from your code or used for multiple projects.
-If you want to install docToolchain separately, you need to create a copy of Config.groovy and
-ConfluenceConfig.groovy for each project. Inside each configuration file you need to specify the input and output
-folders and files.
+In addition to `Config.groovy` there is also a `scripts/ConfluenceConfig.groovy` file. If you are not using Confluence
+you can remove it. If you use Confluence, then you need to open this file and adapt to your environment.
+You can also create multiple copies of that file. For example you can have `ConfluenceConfig.groovy` for publishing
+official pages, and `MyConfluenceConfig.groovy` with a different Confluence space for reviews.
```

Comment:
what happened to the `Custom configuration` section?

Reply:
It's the default configuration now - it's up to the user if they wanted to use directory inside docToolchain, but honestly I don't see why they would. Then each upgrade of docToolchain gets more complicated.